airflow/providers/amazon/aws/operators/sagemaker.py

```diff
@@ -608,3 +608,26 @@ def _check_if_job_exists(self) -> None:
                 raise AirflowException(
                     f'A SageMaker training job with name {training_job_name} already exists.'
                 )
+
+
+class SageMakerDeleteModelOperator(BaseOperator):
```

Comment:
Why this operator inherit from `BaseOperator` and not from `SageMakerBaseOperator` ?

Reply:
Done the changes. I implemented BaseOperator because with SagemakerBaseOperator , we have to pass config dictionary but delete model call just need ModelName as argument but to ensure consistency with other Sagemaker Operator I have made necessary. Please check and resolve. Thanks